src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -284,6 +285,7 @@ def test_daos_oid_allocator(self):
         """
         self.run_subtest()
 
+    @skipForTicket("DAOS-7305")
```

Comment:
We probably do not want to use skipForTicket on variants of daoscoretest. 
Instead, use skip() in C code to skip the subtest that is timing out.
And the issue might be a duplicate of DAOS-7347

Reply:
daoscoretest seems to be a wrapper for the daos tests in C, the skip can be in any place and we get the same results in CI. Give the number 7305 and 7347, 7347 is actually the duped ticket.
Please merge

Reply:
There are 55 subtests in checksum. Not sure if the timeout is consistently happening in a specific subtest since the occurrence is low, only once on 1.2 I think.  I don't think it is worth using skipForTicket to skip all 55 subtests. 
I closed DAOS-7305 as a duplicate  because it never went to triage.
When opening a bug ticket, do not assign it unless you are going to fix the issue. Else, the ticket will not be triaged and assigned to the right owner.

Reply:
What is the proposed path? Re-run the PRs until the failure is not present?

Reply:
I think that depends on the PR. If there are changes in common util files, especially ones related to daoscoretest, yes, I would prefer a rerun.
If and when daos test checksum fails, please increase the number of occurrence in the ticket. The frequency of failure does influence how/when we want to disable a test and the urgency of the ticket to be fixed.

Reply:
This PR is not changing common util files, in fact Im considering the change as doc only. There is a need to update automatically the frequency of failures.

Reply:
Since there was no ticket I opened 7305. 
When 7305 was closed in favor of 7347 the occurrence number was not updated.

Reply:
To avoid any further issues with daos core test, the skip was removed, in case of failure the PR will simply re-run again

Reply:
No.  This is not acceptable.  Tests with known failures MUST be skipped either in code or in the skip-list.  Requiring people to re-run PRs not only increases our CI load unnecessarily but increases user frustration with CI.  I will be changing my +1 to a -1 on this point.

Reply:
This PR is not unskipping a known failure. 
If we can pinpoint the subtest that is failing in checksum, we can skip it in a separate PR.